README.md

```diff
@@ -15,30 +15,30 @@ zip codes, dates of birth, facial features on structural scans.
 
 The processing pipeline to analyze this dataset is available in the [spine-generic repository](https://github.com/sct-pipeline/spine-generic).
 
-## Download
-
-To download the latest version of this dataset, you have two options:
+The entire dataset is about **10GB**.
 
-### Download zip package (recommended)
+## Download
 
-If you are only planning on using the dataset for processing, you can download the latest version as a zip package:
+We are using a tool to manage large datasets alled `git-annex`. To download, dataset, you need to have `git` installed, and also [install `git-annex`](https://git-annex.branchable.com/install/) *at version 8*. Then run:
 
 ~~~
-curl -o spinegeneric.zip -L https://github.com/spine-generic/data-multi-subject/archive/master.zip
-unzip spinegeneric.zip
+git clone https://github.com/spine-generic/data-multi-subject &&
+cd data-multi-subject &&
+git annex init &&
+git annex get
 ~~~
 
-### Clone the repository (slower)
+You may **substitute** `git annex get` with more specific commands if you are only interested in certain subjects. For example:
+
+```
+get annex get sub-nwu01/ sub-nwu03/ sub-nwu04/ sub-oxfordFmrib04/ sub-tokyoSkyra*/
```

Comment:
easier to type
```suggestion
get annex get sub-nwu01 sub-nwu03 sub-nwu04 sub-oxfordFmrib04 sub-tokyoSkyra*
```

Reply:
I read the `/`s as implying "folder". Also, they show up automatically if you use tab-completion.

Reply:
^ i'm fine with that